nevergrad/benchmark/test_xpbase.py

```diff
@@ -29,7 +29,7 @@ def test_run_artificial_function() -> None:
     summary = xp.run()
     assert summary["elapsed_time"] < .5  # should be much faster
     np.testing.assert_almost_equal(summary["loss"], 0.00078544)  # makes sure seeding works!
-    testing.assert_set_equal(summary.keys(), DESCRIPTION_KEYS)
+    testing.assert_set_equal(summary.keys(), DESCRIPTION_KEYS | {"split"})
```

Comment:
I have so much no idea what I'm doing here.

Reply:
You have added a descriptor by adding a parameter, so the tests noticed it. Add "split" in  DESCRIPTION_KEYS directly instead of adding it here and above and below please